ts/demo-functions/src/expand_team_cr.ts

```diff
@@ -25,10 +25,10 @@ export async function expandTeamCr(configs: Configs) {
   // For each 'Team' custom resource in the input:
   // 1. Generate a per-enviroment Namespace.
   // 2. Generate RoleBindings in each Namespace.
-  configs.get(isTeam).forEach(team => {
+  configs.get(isTeam).forEach((team) => {
```

Comment:
Why is there style-related churn? Isn't this enforced by 'npm run lint/format'

Reply:
Prettier changed the default value of arrowParens https://prettier.io/blog/2020/03/21/2.0.0.html

Reply:
That makes me lose confidence in prettier, the whole point of style guides is to keep things consistent, but w/e.